js/index.js

```diff
@@ -1,8 +1,27 @@
 import("../pkg").then(async fluvioWasm => {
   var Fluvio = fluvioWasm.Fluvio;
   var Offset = fluvioWasm.Offset;
-  const fluvio = await Fluvio.connect("ws://localhost:3000")
-  const fluvio_2 = await Fluvio.connect("ws://localhost:3000")
+
+  const fluvio = await Fluvio.connect("ws://192.168.49.2:3030/api/v1/websocket/", "9003", "YOUR TOKEN HERE");
```

Comment:
Should not be hard-coded

Reply:
Yeah, these should be part of some integration suite.